core/domain/subscription_services_test.py

```diff
@@ -56,6 +57,20 @@ def setUp(self):
         self.viewer_id = self.get_user_id_from_email(self.VIEWER_EMAIL)
         self.owner_2_id = self.get_user_id_from_email(self.OWNER_2_EMAIL)
 
+    def _get_all_subscribers_of_creators(self, user_id):
```

Comment:
Okay this is odd. When I try this method, I find that subscriptions_model.subscriber_ids doesn't get updated. On the other hand _get_all....() returns the correct list. subscriptions_model.subscriber_ids also returns the correct value but  first I have to again call the get function like this - `subscriptions_model = user_models.UserSubscriptionsModel.get(
            USER_ID, strict=False)`. After this subscriptions_model.subscriber_ids returns the correct value. Calling this line everytime leads to redundancy which is why I believe we made the above function in the first place. Besides the above function does nothing but gets all the subscribers of the creator. I don't think it's functionality will change in the future. What do you have to say?